src/main/resources/com/smartcodeltd/jenkinsci/plugins/buildmonitor/BuildMonitorView/configure-entries.jelly

```diff
@@ -11,6 +11,15 @@
       <f:option value="2" selected="${it.statusFilter==false}">${%Disabled jobs only}</f:option>
     </select>
   </f:entry>
+  <f:entry title="${%Show builds are being run by timer}">
```

Comment:
I really need to put a stronger warning into this file. Please move BM-specifics to the bottom of the file. One day we'll have to cut'n'paste to main body again from an updated Jenkins baseline.